stock_indicators/indicators/common/candles.py

```diff
@@ -71,7 +67,12 @@ def is_bearish(self) -> bool:
         return self.Close < self.Open
 
 
-CandleProperties = generate_cs_inherited_class(_CandleProperties, CsCandleProperties, "CandleProperties")
+class CandleProperties(generate_cs_inherited_class(_CandleProperties, CsCandleProperties)):
+    """
+    A wrapper class for `CsCandleProperties`, which is an extended version of `Quote`.
```

Comment:
The concept of _wrapper_ is more of a coding convention.  Do you think it makes sense to simply say:

> An extended version of `Quote` that contains additional calculated properties.